apiary.apib

```diff
@@ -1157,7 +1157,8 @@ Further information can be found in the [Developers Documentation](https://devel
 ### Data preview [GET]
 
 Returns up to 1000 rows from table in CSV (RFC) format. Rows can be filtered by various filters described below.
-For full table download please read more in [Asynchronous Unload documentation](#reference/tables/unload-data-asynchronously)
+For full table download please read more in [Asynchronous Unload documentation](#reference/tables/unload-data-asynchronously).
+For the description of filters, see [Asynchronous Unload documentation](#reference/tables/unload-data-asynchronously).
```

Comment:
Popravdě tohle aktuálně neodpovídá. Exporty pro async unload podporují více filtrů pomocí `&whereFilters[0]...`. To mapping do workaspaců podle mě nepodporuje.

Reply:
to se ale tyka Data preview

Reply:
jo takhle. Jak se na slacku probíraly workspace, tak jsem si toho nevšimnul. Tak pak ok.